package.json

```diff
@@ -118,12 +118,12 @@
     "lint-fix": "eslint --fix src",
     "precommit": "lint-staged && npm run test",
     "prefast-test": "npm run prepublish",
-    "prepublish": "npm run package",
+    "prepare": "npm run package",
     "serve": "npm run live-serve",
     "test": "jest",
     "test:debug": "node --inspect node_modules/.bin/jest --watch --runInBand",
     "view-cover": "npm run cover -- --report=html && opn ./coverage/index.html",
-    "package": "npm-run-all clean lint test build build:prod",
```

Comment:
Why was lint and test removed?

Reply:
I think `npm install` runs often, but it's slow to always run things like lint and tests.

In this PR, I intend to execute it automatically by github action, so I think that it will be possible to develop at high speed by confirming success there.

Reply:
> Why was lint and test removed?

@ajbogh hi, revert it. please review this PR. thank you :)